package.json

```
@@ -1,39 +1,15 @@
 {
-	"name": "novastar-controller",
-	"legacy": [
-		"novastar-vx4s"
-	],
-	"version": "1.0.8",
+	"version": "2.0.0",
```

name is mandatory and can't be omitted
```suggestion
       "name": "novastar-controller",
	"version": "2.0.0",
```